src/app/api/v1/webhooks/stripe/connect/route.ts

```diff
@@ -0,0 +1,61 @@
+import to from 'await-to-js'
+import { headers } from 'next/headers'
+import { NextResponse } from 'next/server'
+
+import { stripe } from '@/server/services/stripe'
+import { logger } from '@/server/lib/logger'
+import { createAdminClient } from '@/utils/supabase/serverAdmin'
+
+const endpointSecret = process.env.STRIPE_CONNECT_WEBHOOK_SECRET
+
+if (!endpointSecret)
+  throw new Error('STRIPE_CONNECT_WEBHOOK_SECRET env var is not set')
+
+export const POST = async (req: Request) => {
```

Comment:
I expect the Stripe payments webhook to follow the same pattern. So tagging #8  here for future reference
- #8